components/nds-core/scss/helpers/_helpers-print.scss

```diff
@@ -0,0 +1,42 @@
+////
+/// @group helpers
+////
+
+/// Styles to hide and show under the print media query
+
+/// Hide the element when printing
+/// @output the CSS to hide on the print stylesheet
+/// @since 1.0.0
+@mixin nds-hide-print {
```

Comment:
Can you apply this to the other places where we're already hiding for print e.g. https://github.com/nice-digital/nice-design-system/blob/v1/components/nds-stacked-nav/scss/stacked-nav.scss#L87-L89 (I think we do it for CTA in page header and there are probably other use cases).